docker/docker-compose.yml

```diff
@@ -0,0 +1,35 @@
+version: "3"
+services:
+
+  clickhouse:
+    image: yandex/clickhouse-server:19.1
```

Comment:
Hey. Don't see a reason to stick to some particular version here. Let it be just `latest`

Reply:
This is the one I tried, there might be breaking changes for *graphouse* for the latest? :thinking:

Reply:
Most probably, not.
And stick to a particular version would cause missed features.